spec/calendar.html

```diff
@@ -283,39 +283,60 @@ <h1>GetTemporalCalendarWithISODefault ( _item_ )</h1>
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-temporal-datefromfields" aoid="DateFromFields">
-      <h1>DateFromFields ( _calendar_, _fields_ [ , _options_ ] )</h1>
+    <emu-clause id="sec-temporal-calendardatefromfields" type="abstract operation">
+      <h1>
+        CalendarDateFromFields (
+          _calendar_: an Object,
+          _fields_: an Object,
+          optional _options_: an Object or *undefined*,
+        ): either a normal completion containing a `Temporal.PlainDate` or an abrupt completion
+      </h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>It calls the given _calendar_'s `dateFromFields()` method and validates the result.</dd>
+      </dl>
       <emu-alg>
-        1. Assert: Type(_calendar_) is Object.
-        1. Assert: Type(_fields_) is Object.
         1. If _options_ is not present, set _options_ to *undefined*.
```

Comment:
Is this not the default behavior? cc @bakkot

Reply:
Nope. That only holds for built-in functions, not spec-internal stuff.

Reply:
Ehhh, I don't think it's worth the cost of the additional complexity in the meta-language.